internal/proxy/usecases/proxy_usecase.go

```diff
@@ -70,6 +73,22 @@ func (p *ProxyUsecase) findInDatabaseIfNotFoundInCache(UUID string) error {
 	return nil
 }
 
+// WriteCachedRequestsRoutine  write metric cache in database
+func (p *ProxyUsecase) WriteCachedRequestsRoutine() {
+	logrus.Info("func WriteCachedRequestsRoutine")
+	allRequests, err := p.lruMetrics.GetAll()
+	if err != nil {
+		logrus.Error("could not init the LRU cache")
+	}
+	logrus.Info("len data", len(allRequests))
+	err = p.metricsRepo.SaveMany(allRequests)
+	if err != nil {
+		logrus.Errorf("could not save metrics in database: %s", err)
+	}
+
```

Comment:
I suggest to add a log like : `logrus.Infof("Successfully saved %s cached metrics in database", len(allRequests))`